From 0820a6e2fb005b5f856dda25e1644f34fc994129 Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Date: Wed, 17 Mar 2010 13:40:41 -0500
Subject: [PATCH 15/37] Fix .man.N targets for AM_SILENT_RULES

Add $(AM_V_GEN) for sed-based rules so they appear as expected with
automake silent rules, and $(AM_V_at) to completely hide cp/ln/rm
commands which are not prone to fail.

Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Reviewed-by: Alan Coopersmith <alan.coopersmith@sun.com>
Reviewed-by: Julien Cristau <jcristau@debian.org>
---
 doc/Makefile.am                  |    4 ++--
 hw/dmx/Makefile.am               |    2 +-
 hw/dmx/config/Makefile.am        |    2 +-
 hw/kdrive/ephyr/Makefile.am      |    4 ++--
 hw/vfb/Makefile.am               |    4 ++--
 hw/xfree86/doc/man/Makefile.am   |    8 ++++----
 hw/xfree86/exa/Makefile.am       |    4 ++--
 hw/xfree86/fbdevhw/Makefile.am   |    4 ++--
 hw/xfree86/utils/gtf/Makefile.am |    4 ++--
 hw/xnest/Makefile.am             |    4 ++--
 hw/xquartz/doc/Makefile.am       |    2 +-
 hw/xwin/Makefile.am              |    8 ++++----
 12 files changed, 25 insertions(+), 25 deletions(-)

diff --git xorg-server-1.7.99.902/doc/Makefile.am xorg-server-1.7.99.902/doc/Makefile.am
index bee64b6..2ff683c 100644
--- xorg-server-1.7.99.902/doc/Makefile.am
+++ xorg-server-1.7.99.902/doc/Makefile.am
@@ -14,9 +14,9 @@ CLEANFILES = $(appman_PROCESSED) $(appman_DATA)
 include $(top_srcdir)/cpprules.in
 
 .man.$(APP_MAN_SUFFIX):
-	cp $< $@
+	$(AM_V_at)cp $< $@
 
-EXTRAMANDEFS = -D__default_font_path__="`echo $(COMPILEDDEFAULTFONTPATH) | sed -e 's/,/, /g'`"
+EXTRAMANDEFS = -D__default_font_path__="`echo $(COMPILEDDEFAULTFONTPATH) | $(SED) -e 's/,/, /g'`"
 
 # Docs about X server internals that we ship with source but don't install
 DEVEL_DOCS = smartsched
diff --git xorg-server-1.7.99.902/hw/dmx/Makefile.am xorg-server-1.7.99.902/hw/dmx/Makefile.am
index fc9fde7..0d06346 100644
--- xorg-server-1.7.99.902/hw/dmx/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/Makefile.am
@@ -109,7 +109,7 @@ MAN_SUBSTS = \
 SUFFIXES = .$(APP_MAN_SUFFIX) .man
 
 .man.$(APP_MAN_SUFFIX):
-	$(SED) $(MAN_SUBSTS) < $< > $@
+	$(AM_V_GEN)$(SED) $(MAN_SUBSTS) < $< > $@
 
 EXTRA_DIST = $(appman_PRE)
 
diff --git xorg-server-1.7.99.902/hw/dmx/config/Makefile.am xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
index 7de745f..25a814e 100644
--- xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
+++ xorg-server-1.7.99.902/hw/dmx/config/Makefile.am
@@ -69,7 +69,7 @@ XORGRELSTRING = @PACKAGE_STRING@
 MAN_SUBSTS = -e 's|__vendorversion__|"$(XORGRELSTRING)" "$(XORGMANNAME)"|' 
 
 .man.$(APP_MAN_SUFFIX):
-	$(SED) $(MAN_SUBSTS) < $< > $@
+	$(AM_V_GEN)$(SED) $(MAN_SUBSTS) < $< > $@
 
 EXTRA_DIST = \
 	$(appman_PRE) \
diff --git xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
index d1dd630..8646703 100644
--- xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
+++ xorg-server-1.7.99.902/hw/kdrive/ephyr/Makefile.am
@@ -89,8 +89,8 @@ appmandir = $(APP_MAN_DIR)
 appman_DATA = Xephyr.$(APP_MAN_SUFFIX)
 
 Xephyr.$(APP_MAN_SUFFIX): Xephyr.man
-	-rm -f Xephyr.$(APP_MAN_SUFFIX)
-	$(LN_S) Xephyr.man Xephyr.$(APP_MAN_SUFFIX)
+	-$(AM_V_at)rm -f Xephyr.$(APP_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) Xephyr.man Xephyr.$(APP_MAN_SUFFIX)
 
 include $(top_srcdir)/cpprules.in
 
diff --git xorg-server-1.7.99.902/hw/vfb/Makefile.am xorg-server-1.7.99.902/hw/vfb/Makefile.am
index a183824..4ac0c8f 100644
--- xorg-server-1.7.99.902/hw/vfb/Makefile.am
+++ xorg-server-1.7.99.902/hw/vfb/Makefile.am
@@ -43,8 +43,8 @@ CLEANFILES = $(appman_PRE) $(appman_DATA)
 SUFFIXES += .$(APP_MAN_SUFFIX) .man
 
 .man.$(APP_MAN_SUFFIX):
-	-rm -f $@
-	$(LN_S) $< $@
+	-$(AM_V_at)rm -f $@
+	$(AM_V_at)$(LN_S) $< $@
 
 EXTRA_DIST = Xvfb.man.pre
 
diff --git xorg-server-1.7.99.902/hw/xfree86/doc/man/Makefile.am xorg-server-1.7.99.902/hw/xfree86/doc/man/Makefile.am
index d8b2aa7..737166b 100644
--- xorg-server-1.7.99.902/hw/xfree86/doc/man/Makefile.am
+++ xorg-server-1.7.99.902/hw/xfree86/doc/man/Makefile.am
@@ -8,12 +8,12 @@ filemandir = $(FILE_MAN_DIR)
 fileman_DATA = xorg.conf.$(FILE_MAN_SUFFIX)
 
 Xorg.$(APP_MAN_SUFFIX): Xorg.man
-	-rm -f Xorg.$(APP_MAN_SUFFIX)
-	$(LN_S) Xorg.man Xorg.$(APP_MAN_SUFFIX)
+	-$(AM_V_at)rm -f Xorg.$(APP_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) Xorg.man Xorg.$(APP_MAN_SUFFIX)
 
 xorg.conf.$(FILE_MAN_SUFFIX): xorg.conf.man
-	-rm -f xorg.conf.$(FILE_MAN_SUFFIX)
-	$(LN_S) xorg.conf.man xorg.conf.$(FILE_MAN_SUFFIX)
+	-$(AM_V_at)rm -f xorg.conf.$(FILE_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) xorg.conf.man xorg.conf.$(FILE_MAN_SUFFIX)
 
 include $(top_srcdir)/cpprules.in
 
diff --git xorg-server-1.7.99.902/hw/xfree86/exa/Makefile.am xorg-server-1.7.99.902/hw/xfree86/exa/Makefile.am
index 9eb2e17..da9b720 100644
--- xorg-server-1.7.99.902/hw/xfree86/exa/Makefile.am
+++ xorg-server-1.7.99.902/hw/xfree86/exa/Makefile.am
@@ -22,7 +22,7 @@ driverman_DATA = exa.$(DRIVER_MAN_SUFFIX)
 CLEANFILES = $(driverman_DATA) exa.man
 
 exa.$(DRIVER_MAN_SUFFIX): exa.man
-	-rm -f exa.$(DRIVER_MAN_SUFFIX)
-	$(LN_S) exa.man exa.$(DRIVER_MAN_SUFFIX)
+	-$(AM_V_at)rm -f exa.$(DRIVER_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) exa.man exa.$(DRIVER_MAN_SUFFIX)
 
 EXTRA_DIST = exa.man.pre
diff --git xorg-server-1.7.99.902/hw/xfree86/fbdevhw/Makefile.am xorg-server-1.7.99.902/hw/xfree86/fbdevhw/Makefile.am
index 6a4a6e4..0663525 100644
--- xorg-server-1.7.99.902/hw/xfree86/fbdevhw/Makefile.am
+++ xorg-server-1.7.99.902/hw/xfree86/fbdevhw/Makefile.am
@@ -22,7 +22,7 @@ driverman_DATA = fbdevhw.$(DRIVER_MAN_SUFFIX)
 CLEANFILES = $(driverman_DATA) fbdevhw.man
 
 fbdevhw.$(DRIVER_MAN_SUFFIX): fbdevhw.man
-	-rm -f fbdevhw.$(DRIVER_MAN_SUFFIX)
-	$(LN_S) fbdevhw.man fbdevhw.$(DRIVER_MAN_SUFFIX)
+	-$(AM_V_at)rm -f fbdevhw.$(DRIVER_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) fbdevhw.man fbdevhw.$(DRIVER_MAN_SUFFIX)
 
 EXTRA_DIST = fbpriv.h fbdevhw.man.pre README
diff --git xorg-server-1.7.99.902/hw/xfree86/utils/gtf/Makefile.am xorg-server-1.7.99.902/hw/xfree86/utils/gtf/Makefile.am
index 3ca568d..d1a6d61 100644
--- xorg-server-1.7.99.902/hw/xfree86/utils/gtf/Makefile.am
+++ xorg-server-1.7.99.902/hw/xfree86/utils/gtf/Makefile.am
@@ -40,5 +40,5 @@ CLEANFILES = $(appman_PRE) $(appman_DATA)
 SUFFIXES += .$(APP_MAN_SUFFIX) .man
 
 .man.$(APP_MAN_SUFFIX):
-	-rm -f $@
-	$(LN_S) $< $@
+	-$(AM_V_at)rm -f $@
+	$(AM_V_at)$(LN_S) $< $@
diff --git xorg-server-1.7.99.902/hw/xnest/Makefile.am xorg-server-1.7.99.902/hw/xnest/Makefile.am
index 877ab8b..666a0f0 100644
--- xorg-server-1.7.99.902/hw/xnest/Makefile.am
+++ xorg-server-1.7.99.902/hw/xnest/Makefile.am
@@ -85,8 +85,8 @@ CLEANFILES = $(appman_PRE) $(appman_DATA)
 SUFFIXES += .$(APP_MAN_SUFFIX) .man
 
 .man.$(APP_MAN_SUFFIX):
-	-rm -f $@
-	$(LN_S) $< $@
+	-$(AM_V_at)rm -f $@
+	$(AM_V_at)$(LN_S) $< $@
 
 relink:
 	$(AM_V_at)rm -f Xnest$(EXEEXT) && $(MAKE) Xnest$(EXEEXT)
diff --git xorg-server-1.7.99.902/hw/xquartz/doc/Makefile.am xorg-server-1.7.99.902/hw/xquartz/doc/Makefile.am
index b812af1..7310de3 100644
--- xorg-server-1.7.99.902/hw/xquartz/doc/Makefile.am
+++ xorg-server-1.7.99.902/hw/xquartz/doc/Makefile.am
@@ -8,7 +8,7 @@ CLEANFILES = $(appman_PROCESSED) $(appman_DATA)
 include $(top_srcdir)/cpprules.in
 
 .man.$(APP_MAN_SUFFIX):
-	cp $< $@
+	$(AM_V_at)cp $< $@
 
 EXTRA_DIST = \
 	Xquartz.man.pre
diff --git xorg-server-1.7.99.902/hw/xwin/Makefile.am xorg-server-1.7.99.902/hw/xwin/Makefile.am
index 41a4515..0577300 100644
--- xorg-server-1.7.99.902/hw/xwin/Makefile.am
+++ xorg-server-1.7.99.902/hw/xwin/Makefile.am
@@ -180,12 +180,12 @@ filemandir = $(FILE_MAN_DIR)
 fileman_DATA = XWinrc.$(FILE_MAN_SUFFIX)
 
 XWin.$(APP_MAN_SUFFIX): XWin.man
-	-rm -f XWin.$(APP_MAN_SUFFIX)
-	$(LN_S) XWin.man XWin.$(APP_MAN_SUFFIX)
+	-$(AM_V_at)rm -f XWin.$(APP_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) XWin.man XWin.$(APP_MAN_SUFFIX)
 
 XWinrc.$(FILE_MAN_SUFFIX): XWinrc.man
-	-rm -f XWinrc.$(FILE_MAN_SUFFIX)
-	$(LN_S) XWinrc.man XWinrc.$(FILE_MAN_SUFFIX)
+	-$(AM_V_at)rm -f XWinrc.$(FILE_MAN_SUFFIX)
+	$(AM_V_at)$(LN_S) XWinrc.man XWinrc.$(FILE_MAN_SUFFIX)
 
 EXTRAMANDEFS = -D__logdir__=$(logdir) -D__sysconfdir__=$(sysconfdir) -D__datadir__=$(datadir)
 
-- 
1.7.0.2

